server/examples/go/hello/http_hello/functions.go

```diff
@@ -21,6 +22,7 @@ func (h *helloapp) GetBindings(w http.ResponseWriter, req *http.Request, claims
 func (h *helloapp) Install(w http.ResponseWriter, req *http.Request, claims *apps.JWTClaims, c *apps.CallRequest) (int, error) {
 	out, err := h.HelloApp.Install(AppID, AppDisplayName, c)
 	if err != nil {
+		log.Printf("err: %#+v\n", err)
```

Comment:
Is this intended? If so, should it be more verbose? ("Error installing Hello App" or something like this)